gymnasium/spaces/sequence.py

```diff
@@ -54,8 +59,14 @@ def is_np_flattenable(self):
 
     def sample(
         self,
-        mask: Optional[Tuple[Optional[Union[np.ndarray, int]], Optional[Any]]] = None,
-    ) -> Tuple[Any]:
+        mask: None
+        | (
+            tuple[
+                None | (np.integer | npt.NDArray[np.integer]),
+                Any | None,
```

Comment:
`Any | None == Any`

Reply:
I find Any | None more descriptive and helpful but will change